health-checker/pom.xml

```diff
@@ -180,12 +186,12 @@
               !ch.qos.logback.classic.*,
               !ch.qos.logback.core.*,
               !io.dropwizard.logging,
-              org.osgi.framework
+              io.dropwizard.*,
+              org.osgi.framework,
+              com.codahale.metrics,
+              io.stargate.core.*
             ]]></Import-Package>
-            <Export-Package><![CDATA[
-              io.stargate.health.metrics.api,
-              com.codahale.metrics
-            ]]></Export-Package>
+            <Export-Package>io.stargate.health</Export-Package>
```

Comment:
Actually this no longer needs to export anything since it's now just a top level http service.

Reply:
good catch! I'll fix it.